steam/client/__init__.py

```diff
@@ -115,6 +118,13 @@ def _bootstrap_cm_list_from_file(self):
             self.cm_servers.merge_list(data['servers'])
             self._cm_servers_timestamp = int(data['timestamp'])
 
+    def _handle_server_list(self, msg):
+        for server in msg.body.servers:
+            if server.server_type not in self.servers:
+                self.servers[server.server_type] = []
```

Comment:
server_type should be mapped through the `EServerType` enum

Reply:
`EServerType` holds 97 entries and the server list received only ever contains `EServerType.CS` servers. If you want I will make that change.

Reply:
If it is only `ContentServers` then it will be better to put that code into a `builtin` or `mixins` class that would handle CDN/SteamPipe.

Reply:
I have removed the server list handling from the current PR. Once I find the time I will put out a PR for a mixin